test/Microsoft.AspNetCore.Http.Abstractions.Tests/PathStringTests.cs

```diff
@@ -16,6 +16,33 @@ public void CtorThrows_IfPathDoesNotHaveLeadingSlash()
             ExceptionAssert.ThrowsArgument(() => new PathString("hello"), "value", "The path in 'value' must start with '/'.");
         }
 
+        [Fact]
+        public void Equals_EmptyPathStringAndDefaultPathString()
+        {
+            // Act and Assert
+            Assert.Equal(PathString.Empty, default(PathString));
```

Comment:
Make sure the different overloads are covered:
Assert.True(PathString.Empty == default(PathString));
Assert.True(PathString.Empty.Equals(default(PathString));

Also add variations with the arguments reversed.